dev/src/components/NavBar.tsx

```diff
@@ -1,49 +1,53 @@
 import { Link } from "react-router-dom"
-import NavBarFooter from "./navbar/NavBarFooter"
-import NavBarCourse from "./navbar/NavBarCourse"
-import { isEnrolled, isVisible } from "../Helpers"
-
+import NavFavorites from "./NavFavorites"
+import NavBarUser from "./navbar/NavBarUser"
 
 //TODO Review the NavBar behaviour.
 const NavBar = (): JSX.Element => {
     const state = useAppState()
+    const actions = useActions()
 
-    const visible = state.enrollments.filter(enrollment => isEnrolled(enrollment) && isVisible(enrollment))
-
-    const courses = visible.map((enrollment) => {
-        return <NavBarCourse key={enrollment.getId()} enrollment={enrollment} />
-    })
+    const onCourseClick  = () => {
+        actions.setActiveFavorite(!state.showFavorites)
+    }
 
     return (
-        <nav className="navigator">
-            <ul key="list" className="SidebarList">
-                <li key="logo" className="logo">
-                    <Link to="/">
-                        QuickFeed
-                    </Link>
-                </li>
-                {!state.isLoggedIn &&
-                    <li>
-                        <a href="/auth/github" style={{ textAlign: "center", paddingTop: "15px" }}>
-                            Sign in with <i className="fa fa-2x fa-github align-middle ml-2" id="github" />
+        <nav className="navbar navbar-expand-lg" style={{backgroundColor: "#222", color: "#d4d4d4"}} id="main" >
+            {!state.showFavorites &&
+            <a className="navbar-brand" style={{marginLeft: "30px", fontWeight: "bold"}}>
+                <Link to="/" style={{ fontWeight: "bold", fontSize: "30px", color: "#d4d4d4"}}>
+                    QuickFeed
+                </Link>
+            </a>
+            }     
+            {!state.isLoggedIn &&
+            <div className="navbar-collapse ml-auto">
+                <ul className="ms-auto ml-auto list-unstyled">
+                    <li className="nav-item">      
+                        <a href="/auth/github" style={{ textAlign: "center", paddingTop: "15px", color: "#d4d4d4", marginLeft: "40px"}}>
+                            Sign in with <i className="fa fa-2x fa-github align-middle ms-auto " id="github"/>
                         </a>
                     </li>
-                }
-
-                {courses}
-                {state.isLoggedIn &&
-                    <li key="all" className="">
-                        <Link to="/courses" className="Sidebar-items-link">
-                            View all courses
-                        </Link>
-                    </li>}
-                <NavBarFooter key="foot" />
+                </ul>  
+            </div>
+            } 
+            {state.isLoggedIn &&
+            <ul className="mr-auto me-auto list-unstyled">
+                <a className="clickable"  onClick={() => { onCourseClick()}} style={{paddingTop: "15px", marginLeft: "10px", fontSize:25}}>☰</a>
```

Comment:
Long line; split at appropriate places... Also, make sure to add space after `onCourseClock()`